deliveries/milestone-delivery-template.md

```diff
@@ -1 +1,76 @@
-# Milestone Delivery 
+# JAM Milestone Delivery 
+
+> ⚠ **Only the GitHub account mentioned on the [JAM Prize Interest Form](https://docs.google.com/forms/d/e/1FAIpQLSdvisUospNv_ZUG-RMgCvPD1gp7rtX80Z65NpaQyvovBEf7Zg/viewform) is allowed to submit milestones.**
```

Comment:
I think we don't need this rule. Theoretically, you can deliver the entire project without ever filling out the form. Maybe we could add it to the declaration part below

Reply:
Noted. I will remove this bit entirely.